readme.txt

```diff
@@ -5,7 +5,7 @@ License: GPLv2 or later
 License URI: http://www.gnu.org/licenses/gpl-2.0.html
 Requires at least: WordPress 4.1
 Tested up to: 5.3
```

Comment:
```suggestion
Tested up to: 5.3.2
```
Though minor versions are automatically updated on the plugin page, update version here itself if tested with the latest patch version, i.e `5.3.2`